x-pack/plugins/uptime/public/routes.tsx

```diff
@@ -170,6 +173,29 @@ export const PageRouter: FC = () => {
     }
   `;
 
+  const { basePath } = useContext(UptimeSettingsContext);
+  const { data } = useSelector(indexStatusSelector);
+  const noDataInfo = !data || data?.docCount === 0 || data?.indexExists === false;
+  const noDataConfig = noDataInfo
+    ? {
+        solution: i18n.translate('xpack.uptime.noDataConfig.solutionName', {
+          defaultMessage: 'Observability',
+        }),
+        actions: {
+          beats: {
+            title: i18n.translate('xpack.uptime.noDataConfig.beatsCard.title', {
+              defaultMessage: 'Add monitors with Heartbeat',
+            }),
+            description: i18n.translate('xpack.uptime.noDataConfig.beatsCard.description', {
+              defaultMessage: 'Use Heartbeat to create uptime monitors.',
+            }),
+            href: basePath + `/app/home#/tutorial/uptimeMonitors`,
+          },
+        },
+        docsLink: docLinks!.links.observability.guide,
+      }
+    : undefined;
+
```

Comment:
Uptime already has an empty/no data screen, if we decides to adapt this new appraoch, we will need to get rid of those views.